en/docs/references/postman-integration.md

```diff
@@ -0,0 +1,129 @@
+    ```
+   
+6. Click **Save**. 
+
+Now you can proceed to create a new workspace to link to your service or select an existing workspace in your Postman account.
+
+## Step 3: Link a workspace to your service
+
+In this step, let's connect a workspace to your service by following this procedure:
+
+1. In the **Postman** right panel that you opened, expand the **Select or create a workspace** field.
+
+    ![Postman right panel](../assets/img/postman/postman-right-panel.png){.cInlineImage-full}
+    
+    As shown above, the drop-down list displays the existing workspaces in your Postman account. You can link your service to an existing workspace or create a new one. In this example, let's create a new workspace.
+    
+2. Click **+ Create New**.
+
+3. Enter a name and a description for the new workspace. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                       |
+    |-----------------|---------------------------------|
+    | **Name**        | `covid-Status`                  |
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+!!! tip
+    If you linked an existing workspace to the service and that workspace has collections that are already created in it, you can skip this step.
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** right panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `collection1`                         |
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** right panel.
+
+## Step 5: Create a request
+
+To create a request, follow this procedure:
+
+1. In the **Postman** right panel, click on the collection that you created. This opens the **Requests** section.
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `test1`)
+
+4. Select the required HTTP method. You can also add the query parameters, the header, and the request body as required.
+
+    In this example, you can select **GET** as the HTTP method and add `country` as a query parameter.
+    
+    ![Postman request](../assets/img/postman/postman-request.png){.cInlineImage-full}
+    
+5. Click **Save**.
+
+## Step 6: Check your Postman account
+
+Sign in to your Postman account and check your workspaces. The workspace you created via Choreo will be available as shown below.
+
+![View workspace in Postman](../assets/img/postman/workspace-in-postman.png){.cInlineImage-full}
+
+Click on the workspace. You can view the collection and the request as follows.
```

Comment:
The documentation is abruptly ended without saying how to test  by sending the request.

Reply:
Added more steps to send a request.